doc/source/mast_data_products/hap-parameters.rst

```diff
@@ -3,10 +3,46 @@ HAP Parameters
 
 .. _hap-parameters:
 
+The HAP parameter files are housed in the drizzlepac/pars/hap_pars directory. There are different json parameter files for the SVM and MVM products in separate directories. The svm_parameter json files are also used for pipeline products.
```

Comment:
FYI: General Comment - These are _configuration_ files for HAP which contain values that initialize variables or instantiate objects in a default manner for reasonable processing of the data.  "JSON" just happens to be the ascii storage format for the files.  They could have been stored as XML or some other format. The important part is they are _configuration_ and are needed at the process startup.  As such, the discussion should talk about them as configuration files rather than JSON files.  Note the astrodrizzle configuration files are "cfg" which just another ascii format file.